bin/fin

```diff
@@ -3088,7 +3092,11 @@ config_generate ()
 {
 	eval $(parse_params "$@")
 	# Allow to defined the default stack file.
-	DOCKSAL_STACK=${DOCKSAL_STACK:-default}
+	if [[ "${ARGV[0]}" != "" ]] && [[ "${ARGV[0]}" =~ ^[a-zA-Z0-9]*$ ]]; then
```

Comment:
Is `[[ "${ARGV[0]}" =~ ^[a-zA-Z0-9]*$ ]]` necessary here?
There is a [[ -f ... ]] check down below anyway.